controller/src/main/java/io/pravega/controller/store/stream/ZKStoreHelper.java

```diff
@@ -31,6 +31,8 @@
 
 @Slf4j
 public class ZKStoreHelper {
+    static final String DELETED_STREAMS_PATH = "/lastActiveStreamSegment/%s";
```

Comment:
Please move this constant to ZkStreamMetadataStore

Reply:
Done.